xmlpoke/xmlpoke.d.ts

```diff
@@ -0,0 +1,45 @@
+// Type definitions for xmlpoke 0.1.12
+// Project: https://github.com/mikeobrien/node-xmlpoke
+// Definitions by: Garth Kidd <https://github.com/garthk/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../xpath/xpath.d.ts" />
+
+declare module XmlPoke { // ghost module
```

Comment:
Is `xmlpoke` intended for use only in Node? If so, there's no point in polluting the global namespace with a namespace named `XmlPoke`. Nest the whole thing inside the `declare module 'xmlpoke'` block.